README.md

```diff
@@ -23,7 +23,7 @@ Include a minified browser bundle directly in your HTML like so:
 ```html
 <script
   src="https://unpkg.com/algosdk@v1.22.0/dist/browser/algosdk.min.js"
```

Comment:
@algolucky Minor _suggestion_:  I think the version bump is missed in highlighted `src` value.  

```suggestion
  src="https://unpkg.com/algosdk@v1.23.2/dist/browser/algosdk.min.js"
```